helm/tableland/values.yaml

```diff
@@ -0,0 +1,306 @@
+fullnameOverride: ""
+image: textile/tableland
+imageTag: "v1.8.1-beta-3"
+imagePullPolicy: "IfNotPresent"
+imagePullSecrets: []
+
+podAnnotations: {}
+resources: {}
+  # requests:
+  #   cpu: "1000m"
+  #   memory: "2Gi"
+  # limits:
+  #   cpu: "1000m"
+  #   memory: "2Gi"
+volumeClaimTemplate:
+  accessModes: ["ReadWriteOnce"]
+  resources:
+    requests:
+      storage: 30Gi
+
+config: 
+  config.json: |-
+    {
+      "Impl": "mesa",
+      "HTTP": {
+        "Port": "8080",
+        "RateLimInterval": "1s",
+        "MaxRequestPerInterval": 10,
+        "ApiKey": "${HTTP_RATE_LIMITER_API_KEY}"
+      },
+      "Gateway": {
+        "ExternalURIPrefix": "https://tableland.network",
+        "MetadataRendererURI": "https://tables.tableland.xyz",
+        "AnimationRendererURI": "https://tables.tableland.xyz"
+      },
+      "DB": {
+        "Port": "5432"
+      },
+      "TableConstraints": {
+        "MaxRowCount": 500000
+      },
+      "QueryConstraints": {
+        "MaxWriteQuerySize": 35000,
+        "MaxReadQuerySize": 35000
+      },
+      "Metrics": {
+        "Port": "9090"
+      },
+      "Log": {
+        "Human": false,
+        "Debug": true
+      },
+      "Analytics": {
+        "FetchExtraBlockInfo": true
+      },
+      "Backup": {
+        "Enabled": true,
+        "Dir": "backups",
+        "Frequency": 240,
+        "EnableVacuum": true,
+        "EnableCompression": true,
+        "Pruning": {
+          "Enabled": true,
+          "KeepFiles": 5
+        }
+      },
+      "TelemetryPublisher": {
+        "Enabled": true,
+        "MetricsHubURL": "https://metricshub-mainnet-mrgr43cf5q-uw.a.run.app",
+        "MetricsHubApiKey": "${METRICS_HUB_API_KEY}",
+        "PublishingInterval": "10s",
+        "ChainStackCollectFrequency": "15m"
+      },
+      "Chains": [
+        {
+          "Name": "Ethereum Mainnet",
+          "ChainID": 1,
+          "Registry": {
+            "EthEndpoint": "wss://eth-mainnet.g.alchemy.com/v2/${VALIDATOR_ALCHEMY_ETHEREUM_MAINNET_API_KEY}",
+            "ContractAddress": "0x012969f7e3439a9B04025b5a049EB9BAD82A8C12"
+          },
+          "EventFeed": {
+            "ChainAPIBackoff": "15s",
+            "NewBlockPollFreq": "10s",
+            "MinBlockDepth": 1,
+            "PersistEvents": true
+          },
+          "EventProcessor": {
+            "BlockFailedExecutionBackoff": "10s",
+            "DedupExecutedTxns": true,
+            "WebhookURL": "https://discord.com/api/webhooks/${VALIDATOR_DISCORD_WEBHOOK_ID}/${VALIDATOR_DISCORD_WEBHOOK_TOKEN}"
```

Comment:
I think this config can be removed, this is more like an internal thing we have to notify us when a txn happens (note: it needs to be removed for every chain)